### DISCUSSION TITLE
Not getting invite link when creating users using `tctl create`

### DISCUSSION THREAD

**Discussion by jaskeerat789** [gravitational/teleport#23168] [Q&A]
I am trying to create user using `tctl create` command but i am not getting any invite link similar to one you get when you use `tctl users add`. I want to manage users using declarative yaml file. Due to this issue, i have to use `tctl users add` and then export created user resource yaml using `tctl get users`. Is there a better way to do this?

**Comment by webvictim** [comment]
As suggested on Slack, you can `tctl create user.yaml` to create the user, then run `tctl users reset <username>` to get a reset link. This is slightly more scalable as it keeps the users declarative while still permitting them to have a unique password/second factor.
